collect_app/src/main/java/org/odk/collect/android/javarosawrapper/FormController.java

```diff
@@ -516,6 +516,9 @@ public boolean saveAnswer(FormIndex index, IAnswerData data) throws JavaRosaExce
         try {
             return formEntryController.saveAnswer(index, data, true);
         } catch (Exception e) {
+            Timber.w("Error saving answer of type %s with ref %s for index %s",
```

Comment:
Does this not need to be `Timber.e`?

Reply:
Oh no looks like I'm wrong (looking at `ApplicationInitializer`). Ignore me. I see now what the diff is.

Reply:
Right, it's a little weird. `w` is for things that we don't want as a top-level issue but that we want to show up in the logs we see in Firebase to provide context. It feels like a little bit of an abuse of `w` but I don't see another way. We definitely don't want to send lower log levels to Firebase. 

Side note, it's not totally clear to me whether the top-level "non-fatal" crashes we see from Firebase are from `Timber.e` or from Firebase automatically intercepting exceptions even if they're caught or possibly both.